pkg/mdformatter/linktransformer/config.go

```diff
@@ -69,14 +77,19 @@ const (
 
 const (
 	gitHubAPIURL = "https://api.github.com/repos/%v/%v?sort=created&direction=desc&per_page=1"
+
+	none   = "None"
+	sqlite = "SQLite"
+
+	defaultValidity = time.Duration(432000000000000)
```

Comment:
Better to convert this duration into something more readable: 
```suggestion
	defaultValidity = 2 * time.Hour
```

Reply:
Could even create a separate constant for the duration of a day (there's no `time.Day`) to make it better.